.github/workflows/buid-test-workflow.yaml

```diff
@@ -0,0 +1,39 @@
+name: build-test-workflow
+on: [push]
+jobs:
+  build-darwin:
+    runs-on: macos-10.15
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: cachix/install-nix-action@v13
+        with:
+          nix_path: nixpkgs=channel:nixos-21.05
+
+      - name: Setup temporary cache
+        run: |
+            mkdir -p $HOME/.cache/bazel
+            mkdir -p $HOME/.cache/repo-cache
```

Comment:
Is this missing the caching configuration? I don't see caches uploaded or downloaded.

Reply:
Hey, from what I've read on comments in CircleCI build, cache was supposed to be "created" a new each run. GH-Actions never keep any data from run, unless specified, so this way I make sure, that it's setup should it be needed, but right now it's not really doing anything :)

Reply:
Ah, IIUC, the note in .circleci/config.yaml only refers to builds in master. Other branches do use caching. If you look at the circleci job in this very same PR, you'll find it takes only 8 minutes vs the 43 minutes of the new build-darwin job.